### ISSUE TITLE
[Provide seminars project] OS Image for training, to speed up things at training

### ISSUE THREAD

**Issue by pdgonzalez872** [erlef/education-wg#16]
@costaraphael @jeffgrunewald

**Comment by pdgonzalez872** [comment]
Hopefully folks can set up VirtualBox and have everything ready to go. As vanilla install as possible, Ubuntu, asdf, VSCode/Vim, be able to run Phoenix. That'd be my mvp. Thoughts?

**Comment by elbrujohalcon** [comment]
I don't think this is really needed, personally… Installing Erlang with kerl is super easy, but I'm not sure for Elixir.
In my experience, for the trainings I conducted, just asking everyone to _come with Erlang/Elixir #.# and your favorite text editor installed on your computer_ is enough, while asking people to have VirtualBox (or other similar tools) and download an image and what not is more _controversial_.

**Comment by pdgonzalez872** [comment]
Fair point @elbrujohalcon. Maybe this is a technical solution to an administrative problem. Maybe we make it a requirement to attend the trainings: get a machine that is able to run Erlang/Elixir/Phoenix, etc.

**Comment by pdgonzalez872** [comment]
@elbrujohalcon after seeing what you will present at the meetup, I thought about this: Maybe we create a little Guide on how to run a hello world in Erlang/Elixir (or use something that already exists). We use that as a requirement/something to do before the presentation happens, to ensure they can follow through with the material. Thoughts?

**Comment by elbrujohalcon** [comment]
@pdgonzalez872 I will present that thing at the meetup just because the members of that meetup are all fairly advanced on other topics. There are not many _newcommers_ there, that's all.
I do think that we should aim at having a certain _order_ of talks/presentations, starting with the basics and progressing slowly into more advanced topics. But I believe we should create them when the need arises. i.e. when there is an actual meetup that wants to receive a talk about those topics. In other words: I would not prepare any slides _in advance_ or _just to fill the gaps_. It won't hurt to have like an index of topics we want to cover… but I would not start preparing materials for each topic until we actually need to deliver that training.
Even so, there are tons of _little guides_ on how to get started with Erlang and even more so for Elixir (e.g. https://howistart.org/). I don't think we should create yet another one. We can put our limited resources into other efforts.

**Comment by pdgonzalez872** [comment]
Perfect! I'll check that guide out, may be exactly what I was looking for.
Thanks Brujo!
